fitbook1/src/components/posts/post/postmodal/editpostmodal/EditPostModal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+import "./editpostmodal.css";
+import PrimaryButton from "../../../../buttons/PrimaryButton";
+import Avatar from "../../../../../assets/images/avatar.svg";
+import Picker from "emoji-picker-react";
+import { FaSmile } from "../../../../../assets/icons/icons";
+import { EditPostModalProps } from "./EditPostModalProps";
+
+const EditPostModal = (props: EditPostModalProps) => {
+  const { setEditPostModalOpen, postContent } = props;
+  const wordsLimit = 250;
+  const [remainingWords, setRemainingWords] = useState(wordsLimit);
+  const [isPostDisable, setPostDisabled] = useState(true);
+  const [isEmojiPickerOpen, setEmojiPickerOpen] = useState(false);
+  const [postText, setPostText] = useState(postContent);
+
+  const onChangeText = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
```

Comment:
Appreciation: good job on assigning what exact is going to be changed

Reply:
Appreciation:  nicely done

Reply:
Thanks Brother😊